<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yandex Maps API</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="map">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>311</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Координаты:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>N/S</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="firstCoord"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>E/W</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="secondCoord"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Масштаб</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="scale"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="getPlace">
       <property name="plainText">
        <string>Введите название объекта</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getResult">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="Scheme">
         <property name="text">
          <string>Схема</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Satellite">
         <property name="text">
          <string>Спутник</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Hybride">
         <property name="text">
          <string>Гибрид</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
